syntax/yagpdbcc/main.vim

```diff
@@ -85,14 +85,12 @@ highlight default link yagpdbccKeyword Keyword
 syntax match yagpdbccDot "\v%(\{\{|\s)\."ms=e
     " Order is key here. If you do the dot later, it takes priority over the
     " generic field and top-level object syntaxes, breaking them.
-syntax match yagpdbccObject "\v>@!\.[[:alnum:]\_]+" nextgroup=yagpdbccField
-syntax match yagpdbccField "\v>\)?\zs\.[[:alnum:]\_]+" nextgroup=yagpdbccField
+syntax match yagpdbccObject "\v[\$\)]@<!>@<!\.[[:alnum:]\_]+"
     " We use \zs here to start the match region, because we can't use a
     " constant offset from either end to do so.
```

Comment:
Perhaps use this comment to explain the regex a bit? I feel like that might be appropriate.

Reply:
Definitely a good point - it's quite messy at the moment.